packages/javascript/js-client-library/src/types.ts

```diff
@@ -136,6 +136,16 @@ export interface AssetGroupTagSelectorNode {
     node_id: string;
     certified: AssetGroupTagCertifiedValues;
     certified_by: string | System | null;
+    properties: Record<string, any>;
+    type: string;
+    id: number;
+}
+
+export interface AssetGroupTagMemberInfo {
```

Comment:
I think this interface might be a duplicate of the one above. Maybe this interface was supposed to provide different information for this endpoint? `/api/v2/asset-group-labels/${assetGroupId}/members/${memberId}`

Reply:
Yeah, we have a couple different fields for member info but I'm unsure if we need the  certified fields?

Reply:
From what I recall, the member info endpoint handler and mocks in this changeset were for use in the SelectorList component. The core piece that needs updating here is that you want to have a selector list that is associated with a particular member. 
```suggestion
export interface AssetGroupTagMemberInfo {
    node_id: string;
    certified: AssetGroupTagCertifiedValues;
    certified_by: string | System | null;
    id: number;
    name: string;
    selectors: AssetGroupTagSelector[]
}
```

This means that the response defined that uses this type should instead be:
`export type AssetGroupMemberInfoResponse = BasicResponse<{ member: AssetGroupTagMemberInfo }>;`

and in the SelectorList.tsx component you will need to access the selectors property of the member info to do the mapping:
`selectorsQuery.data.selectors.map((selector, index) => {...`